Automattic/jetpack
# Replace yarn 1 with pnpm

#### Changes proposed in this Pull Request:

Replace yarn 1 with pnpm.

A few side effects:

* Update a bunch of indirect dependencies.
* Add some missing dependencies.
* Move the js-test-runner into its own "project", like the Jetpack CLI, so we can remove its deps from the monorepo-root package.json.
* Deprecated a bunch of scripts in the monorepo-root package.json.

#### Jetpack product discussion


p9dueE-2R4-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






See [docs/yarn-upgrade.md](https://github.com/Automattic/jetpack/blob/feature/replace-yarn-1-with-pnpm/docs/yarn-upgrade.md) before you start, you'll probably want to do some cleanup before checking this out.

* Try everything, see if it still works.
